doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -655,6 +655,12 @@ Modem libraries
 
   * :ref:`nrf_modem_lib_readme` library:
 
+    * Added:
+
+      * The kconfig option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BITRATE` option to enable the :c:func:`nrf_modem_lib_trace_bitrate_get` function that retrieves modem trace backend bitrate statistics.
+      * The kconfig option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BITRATE_LOG` option to enable logging of the modem trace backend bitrate.
+      * The kconfig option :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BITRATE_LOG_PERIOD_MS` to set the period value of the modem trace backend bitrate logging.
```

Comment:
```suggestion
      * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BITRATE` Kconfig option to enable the :c:func:`nrf_modem_lib_trace_bitrate_get` function that retrieves modem trace backend bitrate statistics.
      * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BITRATE_LOG` Kconfig option to enable logging of the modem trace backend bitrate.
      * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BITRATE_LOG_PERIOD_MS` Kconfig option to set the period value of the modem trace backend bitrate logging.
```
The `CONFIG_NRF_MODEM_LIB_TRACE_BITRATE_LOG_PERIOD_MS` Kconfig option, like the other Kconfig options, could be mentioned in the [nrf_modem_lib.rst](https://github.com/nrfconnect/sdk-nrf/pull/8702/files#diff-cbb37da4ceb2d24e21bdad1441e3c33e5bfad3377b5bcf23ab0a9f315d933110).

Reply:
It is not strictly required to mention that Kconfig option in the documentation (as for other similar time-related Kconfig options in the library).